src/duckling/game/drawable/container-drawable.component.ts

```diff
@@ -57,6 +59,17 @@ export class ContainerDrawableComponent {
     @Input() containerDrawable : ContainerDrawable;
     @Output() drawableChanged = new EventEmitter<ContainerDrawable>();
 
+    onChildDrawableCloned(newDrawables : Drawable[]) {
```

Comment:
Going to make this an issue because there are more ways we let keys clash and it should all be fixed in one go.